sdk-reference/mobile/react-native.mdx

```diff
@@ -15,6 +15,74 @@ First, install the Rownd SDK for React Native.
 npm install @rownd/react-native
 ```
 
+### Expo development
+
+1. Add `@rownd/native` as a plugin to your `app.json` file.
+
+```json
+{
+  "expo": {
+    "plugins": ["@rownd/react-native"]
+  }
+}
+```
+
+2. Install [Expo BuildProperties](https://docs.expo.dev/versions/latest/sdk/build-properties/) to set iOS/Android versions
+
+```sh
+npx expo install expo-build-properties
+```
+
+3. Add `expo-build-properties` as a plugin to your `app.json` file. Ensure the Sdk versions match or are above provided iOS/Android versions.
+
+```json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "android": {
+            "minSdkVersion": 26
+          },
+          "ios": {
+            "deploymentTarget": "14.0"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+4. Install Sodium (iOS Framework). [Step 2 for iOS](#ios)
```

Comment:
Is this possible without "ejecting" in an Expo project? Is the podfile available to edit without it getting overwritten?

Reply:
As far as I understand the @rownd/react-native SDK only works when "ejected". It also doesn't look like we could use an expo plugin to modify the podfile: https://docs.expo.dev/config-plugins/development-and-debugging/#modify-ios-podfile